test/integration/xWELL/xWellRouter.t.sol

```diff
@@ -222,4 +240,59 @@ contract xWellRouterTest is Test, ChainIds {
         );
         return mintAmount;
     }
+
+    function testBridgeToBaseFailsInsufficientGlmr() public {
+        vm.expectRevert("xWELLRouter: insufficient GLMR sent");
+        router.bridgeToBase{value: 1}(0);
+    }
+
+    function testBridgeToBaseFailsRefund() public {
+        uint256 mintAmount = xwell.buffer(address(wormholeAdapter));
+
+        uint256 startingXWellBalance = xwell.balanceOf(address(this));
+        uint256 startingXWellTotalSupply = xwell.totalSupply();
+        uint256 startingBuffer = xwell.buffer(address(wormholeAdapter));
+
+        deal(address(well), address(this), mintAmount);
+
+        uint256 bridgeCost = router.bridgeCost() * 2; /// a little extra
+        vm.deal(address(this), bridgeCost);
+
+        uint256 startingWellBalance = well.balanceOf(address(this));
+        uint256 startingLockboxWellBalance = well.balanceOf(address(lockbox));
+
+        well.approve(address(router), mintAmount);
+
+        fallbackReverts = true;
+        vm.expectRevert("xWELLRouter: failed to refund excess GLMR");
+        router.bridgeToBase{value: bridgeCost}(mintAmount);
+    }
+
+    function testBridgeToBaseSucceedsNoRefund() public {
```

Comment:
could we add a test where fallback reverts is set to false and the caller sends a little extra gmlr?

Reply:
never mind, testBridgeOutToSuccess cover this